app/models/translation.rb

```diff
@@ -10,6 +10,19 @@ class Translation < ActiveRecord::Base
 
   @@root_folder_name = 'public/srt/'
 
+  # These are not hashes in reverse order; STATUS_HASH maps from statuses in the database
+  # to status symbols with meaning. POINTS_HASH maps from these symbols to their point value.
```

Comment:
Works. I was thinking more along the lines of descriptions for each hash, but this works.